src/App.css

```diff
@@ -72,3 +72,9 @@ body {
   margin-bottom: 10px;
   background-color: black;
 }
+
+.py-3 {
+  position:absolute;
+  bottom:0;
+  width:100%;
+ }
```

Comment:
Hi @abdiG: Thanks for making this PR. I am more than happy to accept this PR with some slight changes:
1) Could you change the `.py-3` class name to something like `fixed-footer` to be more meaningful. `.py-3` is used by bootstrap to add:

```css
padding-top: 3(units);
padding-bottom: 3(units);
```
2) Add spacing between the css property and value as shown in the above classes:

```css
  position: absolute;
  bottom: 0;
  width: 100%;
```
3) Prettify the Footer component as it was before. Is the pre-commit hook not running for you? The hook should prettify your code before it can be committed.

I would also suggest a change in the color of the text, but I believe that's being addressed in #70.